rubble/src/link/responder.rs

```diff
@@ -75,21 +75,26 @@ impl<C: Config> Responder<C> {
                 }
                 Pdu::DataStart { message } => {
                     info!("L2start: {:?}", HexSlice(message));
-                    this.l2cap().process_start(message)
+                    this.l2cap_tx().process_start(message)
                 }
                 Pdu::DataCont { message } => {
                     info!("L2cont {:?}", HexSlice(message));
-                    this.l2cap().process_cont(message)
+                    this.l2cap_tx().process_cont(message)
                 }
             })
         })
     }
 
-    /// Obtains access to the L2CAP instance.
-    pub fn l2cap(&mut self) -> L2CAPStateTx<'_, C::ChannelMapper, ConfProducer<C>> {
+    /// Obtains access to the L2CAP tx instance.
+    pub fn l2cap_tx(&mut self) -> L2CAPStateTx<'_, C::ChannelMapper, ConfProducer<C>> {
         self.l2cap.tx(&mut self.tx)
     }
 
+    /// Provides mutable access to the L2CAP state.
+    pub fn l2cap(&mut self) -> &mut L2CAPState<C::ChannelMapper> {
```

Comment:
`L2CAPStateTx` already derefs to `L2CAPState`, so I don't really see why this method is necessary